test/e2e/framework/util.go

```diff
@@ -1246,33 +1257,63 @@ func WaitForPodNoLongerRunningInNamespace(c clientset.Interface, podName, namesp
 }
 
 func WaitTimeoutForPodNoLongerRunningInNamespace(c clientset.Interface, podName, namespace, resourceVersion string, timeout time.Duration) error {
-	w, err := c.Core().Pods(namespace).Watch(metav1.SingleObject(metav1.ObjectMeta{Name: podName, ResourceVersion: resourceVersion}))
-	if err != nil {
-		return err
+	return wait.PollImmediate(10*time.Millisecond, timeout, podCompleted(c, podName, namespace))
```

Comment:
Removed those unused `resourceVersion`, and use `Poll` defined in e2e framework (2s) instead of 10ms:

```go 
// How often to Poll pods, nodes and claims.
Poll = 2 * time.Second
```